Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -84,6 +84,7 @@ static float fractionPixels(float pixels) {
 
 static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthFunc, void* widthParam, double offset, CTLineRef* outLine) {
```

Comment:
This is the entry point for NSLayout*

---
In reply to: [77757993](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77757993) [](ancestors = 77757993)